src/libraries/System.DirectoryServices/src/System/DirectoryServices/ActiveDirectory/DirectoryContext.cs

```diff
@@ -555,7 +555,7 @@ internal static string GetLoggedOnDomain()
         {
             string? domainName = null;
 
-            NegotiateCallerNameRequest requestBuffer = default;
+            Interop.Secur32.NegotiateCallerNameRequest requestBuffer = default;
             int requestBufferLength = (int)Marshal.SizeOf(requestBuffer);
```

Comment:
Since this is a blittable type, you can use `sizeof()`.

Reply:
Done in 285bbfa58772b9dfaff8ae3c6aae9a4ec1bc5dc6